template/home.html

```diff
@@ -24,7 +24,9 @@ <h1 class="text-center site-title">doc.crds.dev</h1>
         const input = useRef(null);
 
         const onSubmit = e => {
-            window.location.assign(`/${url}`);
+            const urlParts = url.split("/");
+            urlParts[0] = urlParts[0].toLowerCase();
+            window.location.assign(`/${urlParts.join("/")}`);
```

Comment:
@marccampbell thanks so much for the fix! This seems like an appropriate short-term solution. The backend / API right now is very much in a fast-written, gross but gets the job done state, which I plan on refactoring into a cleaner modular structure in the future. For this workaround, how would you feel about just calling `toLowerCase()` on the whole url rather than just the first section? In practice, it is fine if other parts are not lowercase from a functionality perspective, but it does feel a little cleaner and requires less code that could potentially break in the future if a different url pattern is introduced.

Reply:
Sure. Happy to make that change. I know that some orgs and repos are created with capital letters, and GitHub retains that case and redirects to the format specified during the repo creation/settings. I didn't know if that's a concern.

If the backend getter is also lowercasing them all, I'll clean this up and update the PR with toLowerCase on the url, which will be a little less weird than the implementation I have now.

Reply:
Hmm that is a fair point. The backend will not automatically lowercase repo names, but we likely should. Let's stick with your solution here and I am going to make some fixes on the backend 👍